### DISCUSSION TITLE
How do I attach an the clangd lsp I downloaded using mason to nvim-cmp?

### DISCUSSION THREAD

**Discussion by zaxarakys** [NvChad/NvChad#2961] [Q&A] [answered]
nvim-cmp only shows these snippet things and not the normal autocorrect one would expect... also it doesnt show that thing in its github page where it shows the completion suggestion in the text... i asked around and people told me i had to attach the lsp.... but i dont know how... any help would be appreciated

**Comment by siduck** [comment, marked as answer]
enable lsp for your filetype  https://nvchad.com/docs/config/lsp/
